planning/mission_planner/src/mission_planner/mission_planner.cpp

```diff
@@ -71,14 +71,15 @@ MissionPlanner::MissionPlanner(const rclcpp::NodeOptions & options)
   tf_buffer_(get_clock()),
   tf_listener_(tf_buffer_)
 {
+  map_odom_topic_ = declare_parameter<std::string>("map_odom_topic", "/localization/kinematic_state");
```

Comment:
Topic names should be re-mapped, not taken as arguments. Please update.

Reply:
Fixed. Thank you.